pkgs/top-level/php-packages.nix

```diff
@@ -829,7 +829,10 @@ in
                               . $dev/include/
       '';
 
-      meta.maintainers = lib.teams.php.members;
+      meta = {
+        description = "PHP upstream extension: ${name}";
+        inherit (php.meta) maintainers homepage license;
+      };
```

Comment:
While I agree with the overall idea, I have two questions:

* is it really a good idea to use PHP's license for an arbitrary extension? If it's forgotten to add one, I'd prefer to have no license specified rather than a wrong one.
* should we merge those defaults with `args` in this function using `lib.recursiveUpdate` then? If e.g. an extension only defines `meta.maintainers`, the other values wouldn't exist due to the use of the `//`-operator.

Reply:
These aren't arbitrary though, it's added in the `mkExtension` function which is only executed on the upstream ones that is built from the php source.

And maintainers won't override for other extensions either for the same reason as above :)

Reply:
> These aren't arbitrary though, it's added in the mkExtension function which is only executed on the upstream ones that is built from the php source. 

I see :+1: 

> And maintainers won't override for other extensions either for the same reason as above :)

My mistake, I misread the expression and wrongly assumed that `args` gets merged into the attr-set in `mkExtension`, sorry for that!